data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -82,12 +83,15 @@ public abstract class DeleteReadTests {
 
   @Parameter protected FileFormat format;
 
+  @Parameter(index = 1)
+  protected int formatVersion;
+
   @Parameters(name = "fileFormat = {0}")
   public static Object[][] parameters() {
     return new Object[][] {
-      new Object[] {FileFormat.PARQUET},
-      new Object[] {FileFormat.AVRO},
-      new Object[] {FileFormat.ORC}
+      new Object[] {FileFormat.PARQUET, 2},
+      new Object[] {FileFormat.AVRO, 2},
+      new Object[] {FileFormat.ORC, 2}
```

Comment:
Same as below, any reason we shouldn't just add some V3 cases here?

Reply:
yep we should, updated